File: ruby/fileupload.rb
#!/usr/bin/env ruby

require 'rubygems'
require 'selenium-webdriver'

capabilities = Selenium::WebDriver::Remote::Capabilities.new
capabilities['os'] = 'Windows'
capabilities['os_version'] = '7'
capabilities['browser'] = 'firefox'

driver = Selenium::WebDriver.for(:remote,
  :url => "http://BS_USERNAME:BS_AUTHKEY@hub.browserstack.com/wd/hub", 
  :desired_capabilities => capabilities)

driver.navigate.to "http://www.fileconvoy.com/"
driver.find_element(:id, "upfile_0").send_keys("C:\\Users\\hello\\url.txt")
driver.find_element(:id, "readTermsOfUse").click
driver.find_element(:name, "form_upload").submit
sleep(5)
puts driver.find_element(:tag_name, "center").text
driver.quit

File: node/fileupload.js
var webdriver = require('selenium-webdriver');

var capabilities = {
  'os' : 'Windows',
  'os_version' : '7',
  'browser' : 'firefox',
  'browserstack.user' : 'BS_USERNAME',
  'browserstack.key' : 'BS_AUTHKEY',
}

var driver = new webdriver.Builder().
  usingServer('http://hub.browserstack.com/wd/hub').
  withCapabilities(capabilities).
  build();

driver.get('http://www.fileconvoy.com/');
driver.findElement(webdriver.By.id('upfile_0')).sendKeys('C:\\Users\\hello\\url.txt');
driver.findElement(webdriver.By.id('readTermsOfUse')).click();
driver.findElement(webdriver.By.name('form_upload')).submit();

driver.sleep(5000).then(function() {
        driver.findElement(webdriver.By.tagName('center')).getText().then(function(text) {
            console.log(text);
            });
        });

driver.quit();
